README.md

```diff
@@ -1 +1,61 @@
-# personal-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# personal-portfolio
+
+> Description the project.
+
+
```

Comment:
- [ Optional ]  A brief description of what this project is about will be a good idea don't you think.